### ISSUE TITLE
Grid Issue

### ISSUE THREAD

**Issue by hinkywin** [SimpleMobileTools/Simple-Calendar#1950]
Since simple calendar has been updated to the new design of the simple mobile tools (no orange bar at the top, etc.), events do not "match" the grid in the weekly view. At the very top, everything seems fine, but as I scroll down, things get messed up: An event starting at 1:00 and lasting one hour looks like it starts at 1:00 and lasts one hour - as expected; however, an event starting at 22:00 and lasting one hour looks like it starts at 22:10 and lasts one hour; also, an event starting at 1:00 and lasting 22 hours looks like it starts at 1:00 and lasts 22:10 hours. Meaning, the events seem to be working with a different grid than the one that is displayed by the app ui (in layman's terms). (For what it's worth: An event starting at 23:00 and lasting one hour looks like it starts at 23:10 and lasts 50 minutes, i.e. it finishes at midnight and does not appear at the very top of the following day.) I took a screenshot and wanted to upload it, but github says that "something went really wrong" and won' let me, so I did my best to explain the issue in words. Hope it makes sense! :)

**Comment by tibbi** [comment]
send a screenshot of it please, such things are easier to see like that

**Comment by tibbi** [comment]
just send it at hello@simplemobiletools.com then

**Comment by tibbi** [comment]
right, I see, will fix it soon
